pyproject.toml

```
@@ -40,6 +40,7 @@ pycairo = [
 	{ version = "^1.19", markers = "sys_platform == 'win32'", source="testpypi" },
 ]
 dataclasses = {version = "^0.7", python = "~3.6"}
+pylint = "*"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.0"
```

Why not this line under dev dependency?
```suggestion

[tool.poetry.dev-dependencies]
pytest = "^6.0"
pylint = "*"
```

---

I added this change in the new commit.